electron_app/src/electron-main.js

```diff
@@ -22,8 +22,11 @@ limitations under the License.
 const checkSquirrelHooks = require('./squirrelhooks');
 if (checkSquirrelHooks()) return;
 
+process.setMaxListeners(0);
```

Comment:
Because with spellchecker we have more listener than before. It starts to give overflows which causes to let things glitch